content/guides/models/tables/visualize-tables.md

```diff
@@ -71,6 +71,71 @@ To view the two tables side-by-side, change the first dropdown from "Merge Table
+2. Click **Add panel** in the top right corner of the workspace.
+3. Select **Query panel**.
+4. Within the query expression editor, select `runs` and press **Enter** on your keyboard.
+5. Click the gear icon to view the settings for the panel.
+6. Set **Render As** selector to **Stepper**.
+7. Set **Stepper Key** to `_step` or the [key to use as the unit]({{< relref "#custom-step-keys" >}}) for the step slider.
+
+The following image shows a query panel with three W&B runs and the values they logged at step 295.
+
+{{< img src="/images/data_vis/stepper_key.png" alt="Query panel with three W&B runs and the values they logged at step 295 using the step slider feature.">}}
+
+Within the W&B App UI you may notice duplicate values for multiple steps. This duplication can occur if multiple runs log the same value at different steps, or if a run does not log values at every step. If a value is missing for a given step, W&B use the closest earlier slider key value that was logged.
+
+### Custom step keys
+
+The step key can be any numeric metric that you log in your runs as the step key, such as `epoch` or `global_step`. When you use a custom step key, W&B maps each value of that key to a step (`_step`) in the run.
+
+This table shows how a custom step key `epoch` maps to `_step` values for three different runs: `serene-sponge`, `lively-frog`, and `vague-cloud`. Each row represents a call to `wandb.log()` at a particular `_step` in a run. The columns show the corresponding epoch values, if any, that were logged at those steps. Some `_step` values are omitted to save space.
+
+The first time `wandb.log` was called, none of the runs logged an `epoch` value, so the table shows empty values for `epoch`. 
+
+| `_step` | vague-cloud (`epoch`) | lively-frog(`epoch`) |  serene-sponge (`epoch`) |
+| ------- | ------------- | ----------- | ----------- |
+| 1 | | |  |
+| 2  |   |   | 1  | 
+| 4  |   | 1 | 2  |
+| 5  | 1 |   |  |
+| 6  |  |   | 3  |
+| 8  |  | 2 | 4  |
+| 10 |  |   | 5  |
+| 12 |  | 3 | 6  |
+| 14 |  |   |  7 | 
+| 15 | 2  |   |  |
+| 16 |  | 4 | 8  | 
+| 18 |  |   | 9  |
+| 20 | 3 | 5 | 10 |
+
+Now, if the slider is set to `epoch = 1`, the following happens:
+
+* `vague-cloud` finds `epoch = 1` and returns the value logged at `_step = 5`
+* `lively-frog` finds `epoch = 1` and returns the value logged at `_step = 4`
+* `serene-sponge` finds `epoch = 1` and returns the value logged at `_step = 2`
+
+If the slider is set to `epoch = 9`:
+
+* `vague-cloud` also doesn't log `epoch = 9`, so W&B uses the latest prior value `epoch = 3` and returns the value logged at `_step = 20`
+* `lively-frog` doesn’t log `epoch = 9`, but the latest prior value is `epoch = 5` so it returns the value logged at `_step = 20`
+* `serene-sponge` finds `epoch = 9` and return the value logged at `_step = 18`
+
+<!-- | | |
+| ---- | ---- |
+| Run History Tables Stepper | |
+| Run History Plots Stepper | |
+| Stepper | | -->
+
 ## Compare artifacts
```

Comment:
Maybe this is my own confusion, but the subsections mention comparing tables when I would expect them to mention comparing artifacts.

Reply:
To make it more clear, it might be useful to say "comparing artifacts in a table".  You can use tables to compare artifacts.

Reply:
Or "Use Tables to compare artifacts"?

Reply:
Jira ticket to update subsections: https://wandb.atlassian.net/browse/DOCS-1487 (assigned to myself)